gensim/test/test_logentropy_model.py

```diff
@@ -19,9 +19,42 @@
 
 
 class TestLogEntropyModel(unittest.TestCase):
+    TEST_CORPUS = [[(1, 1.0)], [], [(0, 0.5), (2, 1.0)], []]
+
     def setUp(self):
         self.corpus_small = MmCorpus(datapath('test_corpus_small.mm'))
         self.corpus_ok = MmCorpus(datapath('test_corpus_ok.mm'))
+        self.corpus_empty = [[()]]
```

Comment:
Empty corpus (no documents) is just `[]`. Why the `[[()]]`?

Reply:
Verbose/explicit representation of a corpus?

Reply:
That doesn't look like a corpus at all. A corpus is a sequence of documents, where a document is a list of 2-tuples.

Reply:
+1